### DISCUSSION TITLE
[Calendar] Make calendar only activate from the top/bottom

### DISCUSSION THREAD

**Discussion by AlecMartin3** [fomantic/Fomantic-UI#1556] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Feature Request

Idk if this is possible already, but I believe there should be a toggle for Calender to make it so that it only shows up in one direction for the popup (and similar in the same way as the popup element)

Make it so this is the only way it works:
![HrBcn6h](https://user-images.githubusercontent.com/43552982/86666052-0f92a800-bfa5-11ea-8dce-30ec9a3595af.png)

And avoid situations similar to this:
![UEHZaex](https://user-images.githubusercontent.com/43552982/86666068-13262f00-bfa5-11ea-98db-a645ec559fca.png)

If there already is a way to do this let me know, thanks

**Comment by lubber-de** [comment, marked as answer]
This is already possible by using the popups setting `forcePosition`
![image](https://user-images.githubusercontent.com/18379884/86720826-b028ac80-c025-11ea-8d9a-b2770229f56e.png)

You can specify this for the calendar by providing `popupOptions`, so

```javascript
$('.ui.calendar').calendar({
  popupOptions: {
    position: 'top left',
    forcePosition: true
  }
});
```
